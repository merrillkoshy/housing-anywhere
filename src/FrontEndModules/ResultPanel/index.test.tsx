import React from "react";
import { shallow } from "enzyme";
import Result from ".";

describe("Result Panel", () => {
	const ResultModule = shallow(<Result />);
	it("renders", () => {
		expect(ResultModule).toMatchSnapshot();
	});
	it("finds the panel section", () => {
		expect(ResultModule.find("section").prop("id")).toEqual("result-panel");
	});
});
