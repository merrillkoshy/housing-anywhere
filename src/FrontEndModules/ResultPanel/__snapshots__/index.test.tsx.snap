// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result Panel renders 1`] = `
<section
  id="result-panel"
>
  <div
    className="d-flex justify-content-center w-100 result-panel"
  />
</section>
`;
